prime-router/src/main/java/fhirengine/translation/hl7/structures/fhirinventory/segment/OBR.java

```diff
@@ -0,0 +1,2752 @@
+ * <li>OBR-36: Scheduled Date/Time + (DTM) <b>optional </b>
+ * <li>OBR-37: Number of Sample Containers * (NM) <b>optional </b>
+ * <li>OBR-38: Transport Logistics of Collected Sample * (CWE) <b>optional repeating</b>
+ * <li>OBR-39: Collector's Comment * (CWE) <b>optional repeating</b>
+ * <li>OBR-40: Transport Arrangement Responsibility (CWE) <b>optional </b>
+ * <li>OBR-41: Transport Arranged (ID) <b>optional </b>
+ * <li>OBR-42: Escort Required (ID) <b>optional </b>
+ * <li>OBR-43: Planned Patient Transport Comment (CWE) <b>optional repeating</b>
+ * <li>OBR-44: Procedure Code (CNE) <b>optional </b>
+ * <li>OBR-45: Procedure Code Modifier (CNE) <b>optional repeating</b>
+ * <li>OBR-46: Placer Supplemental Service Information (CWE) <b>optional repeating</b>
+ * <li>OBR-47: Filler Supplemental Service Information (CWE) <b>optional repeating</b>
+ * <li>OBR-48: Medically Necessary Duplicate Procedure Reason (CWE) <b>optional </b>
+ * <li>OBR-49: Result Handling (CWE) <b>optional </b>
+ * <li>OBR-50: Parent Universal Service Identifier (CWE) <b>optional </b>
+ * <li>OBR-51: Observation Group ID (EI) <b>optional </b>
+ * <li>OBR-52: Parent Observation Group ID (EI) <b>optional </b>
+ * <li>OBR-53: Alternate Placer Order Number (CX) <b>optional repeating</b>
+ * <li>OBR-54: Parent Order (EIP) <b>optional </b>
+ * </ul>
+ */
+@SuppressWarnings("unused")
+public class OBR extends AbstractSegment {
+
+  /**
+   * Creates a new OBR segment
+   */
+  public OBR(Group parent, ModelClassFactory factory) {
+    super(parent, factory);
+    init(factory);
+  }
+
+  private void init(ModelClassFactory factory) {
+    try {
+      this.add(SI.class, false, 1, 4, new Object[]{getMessage()}, "Set ID - OBR");
+      this.add(EI.class, false, 1, 0, new Object[]{getMessage()}, "Placer Order Number");
+      this.add(EI.class, false, 1, 0, new Object[]{getMessage()}, "Filler Order Number");
+      this.add(CWE.class, true, 1, 0, new Object[]{getMessage()}, "Universal Service Identifier");
+      this.add(NULLDT.class, false, 1, 0, new Object[]{getMessage()}, "Priority");
+      this.add(NULLDT.class, false, 1, 0, new Object[]{getMessage()}, "Requested Date/Time");
+      this.add(DTM.class, false, 1, 0, new Object[]{getMessage()}, "Observation Date/Time #");
+      this.add(DTM.class, false, 1, 0, new Object[]{getMessage()}, "Observation End Date/Time #");
+      this.add(CQ.class, false, 1, 0, new Object[]{getMessage()}, "Collection Volume *");
+      this.add(XCN.class, false, 0, 0, new Object[]{getMessage()}, "Collector Identifier *");
+      this.add(ID.class, false, 1, 1, new Object[]{getMessage(), Integer.valueOf(65)}, "Specimen Action Code *");
+      this.add(CWE.class, false, 1, 0, new Object[]{getMessage()}, "Danger Code");
+      this.add(ST.class, false, 1, 0, new Object[]{getMessage()}, "Relevant Clinical Information");
+      this.add(DTM.class, false, 1, 0, new Object[]{getMessage()}, "Specimen Received Date/Time");
```

Comment:
Should this be `NULLDT` too? I'm seeing it as backwards compatible in v2.5.1 caristix, then deprecated in NIST, RADx, and v2.7+ caristix

Reply:
For maximum backwards compatibility we are keeping deprecated fields with their original classes rather than using `NULLDT`. The latest updates to this PR bring forward some 2.5.1 definitions that were replaced with `NULLDT` in the 2.7 classes.